src/impi/core.cljs

```diff
@@ -270,6 +322,7 @@
 (def recreate-keys
   #{:pixi.renderer/transparent?
     :pixi.object/type
+    :pixi.graphics/shape
```

Comment:
Typo? This should be `:pixi.graphics/shapes`, perhaps?

But even then I don't think that changing the shapes in a graphics object should recreate the graphics object. This line should just be removed, unless I've missed something?